cadastre/cadastre_export.py

```diff
@@ -148,7 +148,7 @@ def set_composer_templates(self, compte_communal):
         Set parameters for given comptecommunal
         """
         # List of templates
-        compte_communal_abrev = compte_communal[9:]
+        compte_communal_abrev = compte_communal[6:]
```

Comment:
```suggestion
        compte_communal_abrev = compte_communal[-6:]
```
Pour récupérer les 6 derniers caractères correspondant au compte communal abrégé.
Quelle solution est la plus adapatée ici @mdouchin ?